FAIMS/FAIMS3
# feat: back button

## JIRA Ticket

[BSS-389](https://jira.csiro.au/browse/BSS-389)

## Description

Added the option to have a back button within the app as well as fixing the current breadcrumb implementation to work on all screens.

## Proposed Changes

- added navigation environment variable to switch between breadcrumbs and back button
- created re-useable transparent button component
- updated breadcrumbs component to correctly work on all screens rather than separate implementations per screen
- moved fixed breadcrumbs and back button component into the layout to be available on all screens
- removed breadcrumbs component from individual screens
- added some string manipulation functions to handle the path to breadcrumbs conversion

## How to Test

1. Add `VITE_NAVIGATION=breadcrumbs` to your `.env` file
2. Navigate away from home page to multiple different screens
3. Ensure both the appearance and function of the breadcrumbs are working correctly
4. Update `.env` to have `VITE_NAVIGATION=back-button` and repeat steps 2 and 3

## Additional Information

## Checklist
